addons/udes_get_info/models/__init__.py

```diff
@@ -1,2 +1,8 @@
 from . import models
 from . import stock_location
+from . import stock_picking_type  # noqa: F401
```

Comment:
what does noqa mean?

Reply:
I asked the same question, it is used for: flake8 (linter) to ignore that we don't use the imports

Reply:
https://flake8.pycqa.org/en/latest/user/violations.html#in-line-ignoring-errors

Reply:
Thanks. I did not know we use it.